test/cri-containerd/container_virtual_device_test.go

```diff
@@ -200,8 +200,8 @@ func getGPUContainerRequestWCOW(t *testing.T, podID string, podConfig *runtime.P
 func Test_RunContainer_VirtualDevice_GPU_LCOW(t *testing.T) {
 	requireFeatures(t, featureLCOW, featureGPU)
 
-	if osversion.Get().Build < 19566 {
-		t.Skip("Requires build +19566")
+	if osversion.Get().Build < 19041 {
```

Comment:
maybe define a `const`?

Reply:
Agreed, slap it in the osversion package

Reply:
Wait hahaha it already exists https://github.com/microsoft/hcsshim/blob/master/osversion/windowsbuilds.go

Reply:
Well in that case, *use* this const 😆

Reply:
This is what I get for going too fast haha